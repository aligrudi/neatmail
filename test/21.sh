# neatmail test
echo "Hello world!" >/tmp/.attachment
./mail pg -n -a /tmp/.attachment $1 | grep -v "^\(Message-ID\|Date\|From \)"
rm /tmp/.attachment
