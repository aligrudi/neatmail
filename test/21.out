From: 
To: 
Subject: 
User-Agent: neatmail (git://repo.or.cz/neatmail.git)
MIME-Version: 1.0
Content-Type: multipart/mixed; boundary=neatmail-boundary

Multi-part MIME message.
--neatmail-boundary
Content-Type: text/plain; charset=utf-8
Content-Transfer-Encoding: 8bit

MAIL BODY...
--neatmail-boundary
Content-Type: application/octet-stream
Content-Disposition: attachment; filename=".attachment"
Content-Transfer-Encoding: base64

SGVsbG8gd29ybGQhCg==
--neatmail-boundary--

